packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.controller.js

```diff
@@ -1,18 +1,23 @@
+import { TRACKING_S3_POLICY_ADD } from '../../users.constants';
+
 export default class PciUsersAddController {
   /* @ngInject */
   constructor(
     $translate,
     CucCloudMessage,
     PciStoragesObjectStorageService,
     $state,
+    atInternet,
   ) {
     this.$translate = $translate;
     this.CucCloudMessage = CucCloudMessage;
     this.PciStoragesObjectStorageService = PciStoragesObjectStorageService;
     this.$state = $state;
+    this.atInternet = atInternet;
   }
 
   $onInit() {
+    this.trackPage(TRACKING_S3_POLICY_ADD);
```

Comment:
**nitpick:** It is better to use `rename` on `atInternet` obj in the state definition. And moreover, if page tracking is added in the controller, we also need to block the default behavior of page tracking with the state name as label by adding `ignore: true` to the `atInternet` obj in state definition.

Reply:
Taken care as per suggestion